src/lists.py

```diff
@@ -52,7 +68,12 @@ def drop(x: LList[T], k: int) -> LList[T]:
     >>> drop(Link(1, Link(2, None)), 1)
     Link(2, None)
     """
-    ...
+    if x is None or x.tail is None:
+        return None
```

Comment:
If `x` has length 1 and I want to drop 0 elements, I expect to get `x` back. But because `x.tail` is `None` in that case, you will return an empty list.